tutorials/server/install-and-configure-samba/install-and-configure-samba.md

```diff
@@ -0,0 +1,113 @@
+---
+id: install-and-configure-samba
+summary: Install and configure Samba to share files across a network.
+categories: server
+tags: tutorial,installation,ubuntu,file server,samba
+difficulty: 1
+status: Published
+published: 2017-12-01
+
+---
+
+# Install and Configure Samba
+
+## Overview
+Duration: 1:00
+
+A Samba file server enables file sharing across different operating systems over a network.
+
+This guide covers the installation and configuration of Samba on Ubuntu.
+
+### What you'll learn
+- How to set up a Samba file server
+
+### What you'll need
+- Ubuntu 16.04 LTS
+- A Local Area Network (LAN) to share files over
+
+If you have everything ready, let's dive straight into the installation process on the next step!
+
+## Installing Samba
+Duration: 1:00
+
+To install Samba, we run:
+```bash
+sudo apt update
+sudo apt install samba
+```
+We can check if the installation was successful by running:
+```bash
+whereis samba
+```
+The following should be its output:
+```bash
+samba: /usr/sbin/samba /usr/lib/samba /etc/samba /usr/share/samba /usr/share/man/man7/samba.7.gz /usr/share/man/man8/samba.8.gz
+```
+
+## Setting up Samba
+Duration: 2:00
+
+Now that Samba is installed, we need to create a directory for it to share:
+```bash
+mkdir /home/username/sambashare/
+```
+The command above creates a new folder `sambashare` in our home directory which we will share later.
+
+The configuration file for Samba is located at `/etc/samba/smb.conf`. To add the new directory as a share, we edit the file by running:
+```bash
+sudo nano /etc/samba/smb.conf
+```
+At the bottom of the file, add the following lines:
+```
+[sambashare]
+    comment = Samba on Ubuntu
+    path = /home/username/sambashare
+    read only = no
+    browsable = yes
+```
```

Comment:
I guess to fit into the beginner territory covered by this tutorial, it's worth saying something like `Press ctrl-x to save and exit from the *nano* text editor editor`.